pkg/driverbuilder/builder/centos.go

```diff
@@ -23,7 +23,6 @@ type centos struct {
 type centosTemplateData struct {
 	commonTemplateData
 	KernelDownloadURL string
-	GCCVersion        string
```

Comment:
Will we eventually be implementing `GCCVersionRequestor` interfaces for all builders? +1  from me for requiring `GCCVersionRequestor` to be defined for all new builders (moving forward, of course).

Reply:
When not defined, `builder.defaultGCCVersion` will give us the GCC version to be used using the kernelrelease.
I think indeed we should need to implement `GCCVersionRequestor` only in very specific cases, ie: when the default algorithm is off for some reasons.